jest-common/src/main/java/io/searchbox/core/search/aggregation/PercentilesAggregation.java

```diff
@@ -23,7 +25,8 @@ public PercentilesAggregation(String name, JsonObject percentilesAggregation) {
 
     private void parseSource(JsonObject source) {
         for (Map.Entry<String, JsonElement> entry : source.entrySet()) {
-            if(!(Double.isNaN(entry.getValue().getAsDouble()))) {
+            // In case of no results, in Elasticsearch the behaviour was to get 'NaN' and in OpenSearch is 'null'
+            if(!entry.getValue().equals(jsonNull) && !(Double.isNaN(entry.getValue().getAsDouble()))) {
```

Comment:
We could do just `instance of`, but I thought it'll be nicer.
That's the `equals()` method of `JsonNull`
![image](https://user-images.githubusercontent.com/64737978/205490515-6a19916d-e15b-4229-8e46-865925e4c4bd.png)

Reply:
i'd compare the `entry` directly to `JsonNull.INSTANCE` which makes it clear that it's a singleton (imo the `jsonNull` variable here is unnecessary and makes it less obvious)

Reply:
agree.. will change